karapace/schema_references.py

```diff
@@ -68,7 +68,7 @@ def from_dict(data: JsonObject) -> Reference:
         return Reference(
             name=str(data["name"]),
             subject=Subject(str(data["subject"])),
-            version=ResolvedVersion(cast(int, data["version"])),
+            version=Version(cast(int, data["version"])),
```

Comment:
Here, we do not use the `Versioner.V()` factory as there was a cyclic dependency between this module and `schema_models`.